oaknational/Oak-Web-Application
# Feat/cur 330/add subject tags for science years 7 8 9

## Description

CUR-330 - https://www.notion.so/oaknationalacademy/Add-subject-tags-for-Science-years-7-8-9-e65d572090ad43e78c5e0a76bf2fae06

This feature adds science subject tabs for KS3 (Years 7, 8 & 9) to allow users to filter and find relevant units.

They have been added in CAT as tags on the units with the category type "Discipline"

An MV was created to bring in the tag information for each unit, and this ticket uses those tags to add the filters/tabs when a unit has those attached.

## How to test

1. Go to https://deploy-preview-2336--oak-web-application.netlify.thenational.academy/teachers/curriculum/science-secondary-aqa/units
2. You should see these tabs on year 7 - All, Biology, Chemistry & Physics
3. Selecting these should filter the units as expected (check expected results in CAT)

## Screenshots

**How it used to look:**

<img width="1256" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/587241/8e83b910-f9c5-47d5-bced-3c2e923c28bb">
 
**How it should now look:**

<img width="1264" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/587241/0148f936-7c94-45a3-a5ff-143d15a0d3b0">

## Checklist
